website/docs/reference/resource-configs/materialize-configs.md

```diff
@@ -5,18 +5,62 @@ id: "materialize-configs"
 
 ## Performance optimizations
 
-### Incremental models
-Materialize, at its core, is a real-time database that delivers incremental view updates without ever compromising on latency or correctness.
-Materialized views are incremental models, defined once. 
+### Clusters
+
+<Changelog>
+
+- **v1.2.0:** Enable the configuration of [clusters](https://github.com/MaterializeInc/materialize/blob/main/misc/dbt-materialize/CHANGELOG.md#120---2022-08-31).
+
+</Changelog>
+
+The [cluster](https://materialize.com/docs/overview/key-concepts/#clusters) that dbt uses to maintain materialized views or indexes can be configured in your [profile](/reference/profiles.yml) for Materialize connections. To override the cluster that is used for specific models (or groups of models), use the `cluster` model configuration.
+
+<File name='my_view_cluster.sql'>
+
+```sql
+{{ config(materialized='materializedview', cluster='not_default') }}
+
+select ...
+```
+
+</File>
+
+<File name='dbt_project.yml'>
+
+```yaml
+models:
+  project_name:
+    +materialized: materializedview
+    +cluster: not_default
+```
+
+</File>
+
+
+**Recommendation** Create a cluster to use for anything built by dbt. This will leave Materialize's default cluster free for responding quickly to `SHOW MATERIALIZED VIEWS` and other diagnostic queries.
+
+### Incremental models: Materialized Views
+
+Materialize, at its core, is a real-time database that delivers incremental view updates without ever compromising on latency or correctness. Use [materialized views](https://materialize.com/docs/overview/key-concepts/#materialized-views) to compute and incrementally update the results of your query.
 
 ### Indexes
 
+<Changelog>
+
+- **v1.2.0:** Enable additional configuration for [indexes](https://github.com/MaterializeInc/materialize/blob/main/misc/dbt-materialize/CHANGELOG.md#120---2022-08-31).
+
+</Changelog>
+
+Indexes assemble and maintain a query’s results in memory within a cluster, which provides future queries the data they need in a format they can immediately use.
```

Comment:
```suggestion
Like in any standard relational database, you can use [indexes](https://materialize.com/docs/overview/key-concepts/#indexes) to optimize query performance in Materialize. Improvements can be significant, reducing response times down to single-digit milliseconds.
```

Reply:
It helps to make the description less Materialize-y and more relatable to dbt users (we also need to make it more relatable in our docs; the suggestion is from a recent attempt to do that in the `Optimization` page).